Docs: Convert Extending and Answers chapters (#4751)
geodynamics/aspect: doc/sphinx/user/extending/plugin-types/vis-postprocessors.md

furthermore not scale very well as the list continues to grow. Consequently,
as with the postprocessors described in the previous section, what *can* be
computed is implemented in a number of plugins and what *is* computed is

> selected in the input parameter file (see {ref}`sec:3.167`).

Reviewer: ```suggestion
selected in the input parameter file (see {ref}`parameters:Postprocess/Visualization`).
```

> :::{note}
> Reference to sec;3.167 is a reference to a parameter file which is not currently set up.
> :::

Reviewer: ```suggestion
```

`aspect::Postprocess::VisualizationPostprocessors`, and is registered using a
macro, here called `ASPECT_REGISTER_VISUALIZATION_POSTPROCESSOR`. Like the
postprocessor plugins, visualization postprocessors can derive from class

> `aspect::Postprocess::SimulatorAccess` if they need to know specifics of the

Reviewer: This was wrong before.
```suggestion
`aspect::SimulatorAccess` if they need to know specifics of the
```
